src/Components/Publishing/Ads/EnabledAd.ts

```diff
@@ -0,0 +1,27 @@
+import { get } from "lodash"
+import { data as sd } from "sharify"
+
+/**
+ * For now ads, served by Hashtag Labs are only to viewable in the following instances:
+ *  1) When the environment is NOT Production (HASHTAG_LAB_ADS_ENABLED set to false)
+ *  2) When a user is an Admin
+ *  3) When a user has been added to the allowlist set via HASHTAG_LAB_ADS_ALLOWLIST
+ */
+
+export const isHTLAdEnabled = () => {
+  const allowedUsers = (sd.HASHTAG_LAB_ADS_ALLOWLIST || "")
+    .split(",")
+    .filter(Boolean)
+  const currentUser = get(sd, "CURRENT_USER.email", "")
```

Comment:
I think it's ok to add `CURRENT_USER` value to `sharify.d.ts` for now. When this test is removed we can delete it.

Reply:
```js
import { get } from 'Utils/get'

const currentUserEmail = get(sd, => sharifyData.CURRENT_USER.email, '')
```

Reply:
Can I do this in a separate PR?

Reply:
Trying to get these ads available on Staging so they can be tested by the folks at HTL.

Reply:
Yup totally 👍